app/controllers/taxons_controller.rb

```diff
@@ -141,6 +143,10 @@ def bulk_update
 
   def publish
     Services.publishing_api.publish(content_id)
```

Comment:
Would it be useful to extract the type of logic in line 145-148 to a separate module/class? It is repeated in quite a few places in the code and might make it difficult to change/delete at a later date.

Reply:
The publish API call for the Brexit taxon happens twice so I don't think it's too repetitive at this stage.  If there was another instance then I'd be tempted to look at extracting it out.  Also, if we lifted 145-148 out in future we would need to remember to put 145 back in anyway.  Hopefully the use of the `brexit_taxon?` method is clear enough for a future developer to know that anything it's wrapped around should be removed.